liota/core/package_manager.py

```diff
@@ -641,9 +659,13 @@ def _package_load(self, file_name, checksum=None, autoload_flag=False,
                         break
                 if (verify_flag == False):
                     log.error("Package %s integrity verification failed" % path_file_ext)
+                    with open(self._rsp_pipe_file, "w+") as fp:
+                        fp.write('Failure\n')
```

Comment:
Same comment about putting protocol strings somewhere global for client/server to share.

Reply:
done the same as above